Makefile

```diff
@@ -29,18 +29,21 @@ GO_BUILD_ARGS = \
 
 ANSIBLE_BASE_IMAGE = quay.io/operator-framework/ansible-operator
 HELM_BASE_IMAGE = quay.io/operator-framework/helm-operator
+CUSTOM_SCORECARD_TESTS_BASE_IMAGE = quay.io/operator-framework/custom-scorecard-tests
 SCORECARD_TEST_BASE_IMAGE = quay.io/operator-framework/scorecard-test
 SCORECARD_TEST_KUTTL_BASE_IMAGE = quay.io/operator-framework/scorecard-test-kuttl
 
 ANSIBLE_IMAGE ?= $(ANSIBLE_BASE_IMAGE)
 HELM_IMAGE ?= $(HELM_BASE_IMAGE)
+CUSTOM_SCORECARD_TESTS_IMAGE ?= $(CUSTOM_SCORECARD_TESTS_BASE_IMAGE)
 SCORECARD_TEST_IMAGE ?= $(SCORECARD_TEST_BASE_IMAGE)
 SCORECARD_TEST_KUTTL_IMAGE ?= $(SCORECARD_TEST_KUTTL_BASE_IMAGE)
 
-ANSIBLE_ARCHES:="amd64" "ppc64le" "arm64"
-HELM_ARCHES:="amd64" "ppc64le" "arm64"
-SCORECARD_TEST_ARCHES:="amd64" "ppc64le" "arm64"
-SCORECARD_TEST_KUTTL_ARCHES:="amd64" "ppc64le" "arm64"
+ANSIBLE_ARCHES:="amd64" "ppc64le" "s390x" "arm64"
+HELM_ARCHES:="amd64" "ppc64le" "s390x" "arm64"
+CUSTOM_SCORECARD_TESTS_ARCHES:="amd64" "ppc64le" "s390x" "arm64"
+SCORECARD_TEST_ARCHES:="amd64" "ppc64le" "s390x" "arm64"
+SCORECARD_TEST_KUTTL_ARCHES:="amd64" "ppc64le" "s390x" "arm64"
```

Comment:
Are we adding s390x back in? @joelanford thoughts? Travis support did say things should be working again.

Reply:
IHMO we need at least add a comment to clarifies that this images are just examples for the doc and are not required for the tool or its features work at all.

Reply:
removed the 390x from the ARCHES to match what is on master right now, also added a comment in the makefile about the scorecard custom image being an example.